conn.go

```diff
@@ -169,9 +171,16 @@ func createConn(nextConn net.Conn, flightHandler flightHandler, handshakeMessage
 	// Handle inbound
 	go c.inboundLoop()
 
-	<-c.handshakeCompleted
-	c.log.Trace("Handshake Completed")
```

Comment:
What do you think of `c.log.Trace(fmt.Sprintf("Handshake Completed %s", c.getConn()))` instead?

Probably a nit, but adding complexity for logging is something would like to avoid. Sorry to be annoying, but I just worry as we grow `pion/dtls` it will be harder for others to jump in

Reply:
Hmm, not sure I follow. do you mean `getConnErr()` or add a new function `getConn()` for the log?

Reply:
sorry yes `getConnErr()` my mistake!

Reply:
Fixed up, does that look right?

Reply:
Oh yea that LGTM! Just nice to not worry about multiple branches